### PR TITLE
Revert "Feature/seed 755 feature section"

### PR DESCRIPTION
**Pull request by Mitso**

### CODE REVIEW

**Comment by Mitso** [comment]
@peterp oops, sorry I merged by a mistake here.
Please review

File: iogt/templates/core/tags/section_listing_homepage.html [hunk 1]
```diff
@@ -11,13 +11,13 @@ <h1 class="heading heading--homepage">{% trans "Recommended" %}</h1>
...
+              <a href="{% pageurl article %}" class="article-list__image">
                 {% image article.image width-290 %}
               </a>
             {% endif %}
-            <h3 class="section-list__title section-list__title--homepage">{{section.title}}</h3>
+            <h3 class="article-list__section">{{section.title}}</h3>
             
-            <a href="{% pageurl article %}">
+            <a href="{% pageurl article %}" class="article-list__description">
               <h2 class="article-list__title article-list__title--hero">{{article.title}}</h2>
```
Comment (peterp) [inline iogt/templates/core/tags/section_listing_homepage.html:21]: please remove article-list__title--hero

File: iogt/templates/core/article_page.html [hunk 2]
```diff
@@ -4,51 +4,51 @@
...
-      {{comment_count}}
-      {% trans "comment" %}{{comment_count|pluralize}}
-    </a>
+
+  {% if self.image %}
+    {% image self.image width-320 class="article__image" %}
   {% endif %}
+  <p class="article__subtitle">{{self.subtitle}}</p>
+
   <article>
```
Comment (peterp) [inline iogt/templates/core/article_page.html:42]: can we just make this a div.article__content

**Comment by Mitso** [comment]
@peterp I've updated  the above.

**Pull request merged**